plugins/header_rewrite/conditions.cc

```diff
@@ -225,8 +223,8 @@ ConditionHeader::append_value(std::string &s, const Resources &res)
     TSDebug(PLUGIN_NAME, "Getting Header: %s, field_loc: %p", _qualifier.c_str(), field_loc);
 
     while (field_loc) {
-      value          = TSMimeHdrFieldValueStringGet(bufp, hdr_loc, field_loc, -1, &len);
-      next_field_loc = TSMimeHdrFieldNextDup(bufp, hdr_loc, field_loc);
+      const char *value = TSMimeHdrFieldValueStringGet(bufp, hdr_loc, field_loc, -1, &len);
+      next_field_loc    = TSMimeHdrFieldNextDup(bufp, hdr_loc, field_loc);
```

Comment:
Why not move the declaration for `next_field_loc` here as well?

Reply:
Sure.